pkgs/by-name/em/emacsclient-commands/package.nix

```
@@ -0,0 +1,28 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, buildGoModule, runCommandLocal }:
+
+buildGoModule rec {
+  pname = "emacsclient-commands";
+  version = "0.0.0-unstable-2023-09-22";
+
+  src = fetchFromGitHub {
+    owner = "szermatt";
+    repo = pname;
```

we prefer to hardcode the repo name because of override interactions with the package.
```suggestion
    repo = "emacsclient-commands";
```